From 1c9f805b5e06a5a0b2bb5223d9eefb3f24dd3ac7 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Thu, 24 Oct 2013 17:10:16 +0800
Subject: [PATCH 39/68] ENGR00284852 Make dropping frame as ALOGV log

Avoid too much log for this frame dropping in WFD user case

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 services/camera/libcameraservice/api1/client2/StreamingProcessor.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/camera/libcameraservice/api1/client2/StreamingProcessor.cpp b/services/camera/libcameraservice/api1/client2/StreamingProcessor.cpp
index 231dc17..bf569ba 100644
--- a/services/camera/libcameraservice/api1/client2/StreamingProcessor.cpp
+++ b/services/camera/libcameraservice/api1/client2/StreamingProcessor.cpp
@@ -691,7 +691,7 @@ status_t StreamingProcessor::processRecordingFrame() {
         }
 
         if ( mRecordingHeapFree == 0) {
-            ALOGE("%s: Camera %d: No free recording buffers, dropping frame",
+            ALOGV("%s: Camera %d: No free recording buffers, dropping frame",
                     __FUNCTION__, mId);
             mRecordingConsumer->releaseBuffer(imgBuffer);
             return NO_MEMORY;
-- 
1.8.0

